### PR TITLE
 Require comments to have text in them

### PR DESCRIPTION
**Pull request by CalvinWu4**
In response to https://github.com/fedora-infra/bodhi/issues/2009

### CODE REVIEW

**Comment by bowlofeggs** [comment]
jenkies test

File: bodhi/server/models.py [hunk 1]
```diff
@@ -2748,6 +2748,9 @@ def comment(self, session, text, karma=0, author=None, anonymous=False,
         """
         if not author:
             raise ValueError('You must provide a comment author')
+            
+        if not text:
+            raise ValueError('You must provide text in comment')
```
Comment (bowlofeggs) [inline bodhi/server/models.py:2753]: As @mattiaverga commented in #2009, we don't necessarily want to restrict users from providing only karma. I suggest changing this to ```if not text and not karma```.

**Review by bowlofeggs — CHANGES_REQUESTED** [review]
We will want tests for this change. You can add tests at ```bodhi/tests/server/test_models.py```.

**Comment by bowlofeggs** [comment]
Please also amend your commit message to sign the Developer Certificate of Origin as documented in Bodhi's [contributor guidelines](https://bodhi.fedoraproject.org/docs/developer/index.html#contribution-guidelines).

File: bodhi/server/models.py [hunk 2]
```diff
@@ -2748,6 +2748,9 @@ def comment(self, session, text, karma=0, author=None, anonymous=False,
         """
         if not author:
             raise ValueError('You must provide a comment author')
+
```
Comment (bowlofeggs) [inline bodhi/server/models.py:2751]: The CI test failure is due to this line containing whitespace:

```
f26	./bodhi/server/models.py:2751:1: W293 blank line contains whitespace
f26	            
f26	^
```

**Comment by bowlofeggs** [comment]
@CalvinWu4 Do you still want to work on this?

**Comment by CalvinWu4** [comment]
I am currently busy with schoolwork at the moment but I may be able to work on this during the summer

**Comment by bowlofeggs** [comment]
OK, I am going to close this pull request for now. Feel free to reopen it when you have time to work on it.

**Pull request closed without merging**